lifechanger/src/app/hero/hero.component.html

```diff
@@ -0,0 +1,18 @@
+<section>
+  <div class="hero__container">
+    <div class="hero__left-side">
+      <h1 class="hero__heading">
+        Hi 👋,<br />
```

Comment:
I think it's better use tags line margin, paddings instead, but it's optional point
 I
![image](https://github.com/TEZv/Become_PRO_with_Angular/assets/34271977/9f0d6c63-5206-471c-a967-455a4cca08ce)

Reply:
Dear Mentor, thank you for reviewing the code.
I replaced the **<br />** with display: block in the .css file.
Please, review the code again🙂